tests/template_tests/test_library.py

```diff
@@ -90,6 +90,15 @@ def test_simple_tag_invalid(self):
         with self.assertRaisesMessage(ValueError, msg):
             self.library.simple_tag('invalid')
 
+    def test_simple_tag_wrapper(self):
+        from functools import lru_cache
+
+        @self.library.simple_tag
+        @lru_cache(32)
+        def func():
+            return ""
+        self.assertIn("func", self.library.tags)
```

Comment:
A test to assert calling the function twice as a template tag is only called once should be added.

Reply:
Instead of a counter-thingy, we can just control if the wrapped tag has `cache_info` attribute. If it has, the operation can be classified as successful